### PR TITLE
Upgrade CANN version to 8.0.RC1.alpha001

### PR DESCRIPTION
**Pull request by hipudding**
Upgrade CANN version to avoid AscendC kernel runtime error.

### CODE REVIEW

**Comment by asmorkalov** [comment]
@fengyuentau Could you take a look?

**Comment by fengyuentau** [comment]
Looks good to me.

---

BTW, @asmorkalov I dont have reviewer permission in this org. I cannot add myself as reviewer.

File: openeuler-github-actions-ascend310--22.03.sp2/Dockerfile [hunk 1]
```diff
@@ -22,22 +22,25 @@ RUN yum install -y \
...
+    ./Ascend-cann-kernels-310p_7.0.0.alpha003_linux.run --quiet --install && \
+    rm -f ./Ascend-cann-kernels-310p_7.0.0.alpha003_linux.run
 
 # Self-hosted runner UID is 1001, matches user id on host.
 RUN groupadd -g 1000 HwHiAiUser && \
     useradd ci -m -s /bin/bash -G users --uid=1001 && \
     usermod -a -G HwHiAiUser ci
 
+# Workaround for AscendC build bug with ninja, remove this line for next cann version.
+RUN sed -i "/\$(MAKE)/d" /usr/local/Ascend/ascend-toolkit/latest/compiler/tikcpp/ascendc_kernel_cmake/function.cmake
```
Comment (fengyuentau) [inline openeuler-github-actions-ascend310--22.03.sp2/Dockerfile]: A month was past. Is there a newer version with this issue addressed?

Comment (hipudding) [reply openeuler-github-actions-ascend310--22.03.sp2/Dockerfile]: Yes, there's a new version now, I will update this dockerfile.

Comment (MengqingCao) [reply openeuler-github-actions-ascend310--22.03.sp2/Dockerfile]: Tests passed, and the latest version (8.0.RC1.alpha001) of CANN can solve this issue.

**Review by fengyuentau — APPROVED** [review]
👍 Thank you!

**Pull request merged**